### ISSUE TITLE
Support constant values

### ISSUE THREAD

**Issue by presidento** [bhch/react-json-form#65]
JSON schema supports constant values: https://json-schema.org/understanding-json-schema/reference/generic.html#constant-values

Until it is implemented, there is a workaround for it by converting `{"const": $value}` in a pre-processing step to

```json
{
    "type": <type of $value>,
    "value": $value,
    "readonly": true
}
```

**Comment by bhch** [comment]
Hi, thanks for opening this issue.

I will implement this feature. But I'm also looking for ideas as to how the UI must behave. Should it has a readonly input? Or a regular editable input?

**Comment by bhch** [comment]
Implemented in 6f987cb02479d5d665fe01448894c61f54837c4e.

Will be released in the next version.
